src/viam/sdk/spatialmath/geometry.hpp

```diff
@@ -23,7 +23,48 @@ class GeometryConfig {
     viam::common::v1::RectangularPrism box_proto() const;
     viam::common::v1::Sphere sphere_proto() const;
     viam::common::v1::Pose pose_proto() const;
-    static GeometryConfig from_proto(viam::common::v1::Geometry proto);
+    static GeometryConfig from_proto(const viam::common::v1::Geometry &proto);
+    static std::vector<GeometryConfig> from_proto(const viam::common::v1::GetGeometriesResponse &proto);
+    void set_coordinates(double x, double y, double z) {
+        x_ = x;
+        y_ = y;
+        z_ = z;
+    }
+    void set_pose(double x, double y, double z) {
+        o_x_ = x;
+        o_y_ = y;
+        o_z_ = z;
+    }
+    void set_radius(double r) {
+        r_ = r;
+    }
+    void set_geometry_type(GeometryType type) {
+        geometry_type_ = type;
+    }
+    void set_orientation_config(OrientationConfig config) {
+        orientation_config_ = config;
+    }
+    void set_label(std::string label) {
+        label_ = label;
+    }
+    std::vector<double> get_coordinates(){
```

Comment:
returning a vector here is a little brittle. There's nothing programmatically enforcing that it be the right length. I think we should probably instead return a tuple to enforce the number of values, or (if you're up for a slightly larger refactor) make a `coordinates` struct and a `pose` struct, and have the argument for the setter/return for the getter be those structs instead. I leave it up to you :)